core/common/src/main/java/alluxio/conf/ReconfigurableRegistry.java

```diff
@@ -46,11 +47,17 @@ public static synchronized boolean unregister(Reconfigurable listener) {
    */
   public static synchronized boolean update() {
```

Comment:
Do we still need to call this method?

Reply:
It is necessary as we have to keep consistent with some old caller